api.planx.uk/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@airbrake/node": "^2.1.7",
-    "@opensystemslab/planx-document-templates": "git://github.com/theopensystemslab/planx-document-templates#main",
+    "@opensystemslab/planx-document-templates": "git://github.com/theopensystemslab/planx-document-templates#e0642be",
```

Comment:
this approach totally works for me for now until we have bigger picture convos about monorepo tooling or a testing module in place like you describe. i'm on board with the wonky workflow in the shorterm, especially as document templates requests hopefully slow down soon and won't be expected to change that often.

i am wondering though if it's worth adding a sentence about this to each the API & Editor READMEs in planx-new? something like: 

- `pnpm i` installs dependencies
  - if you need to test or pull new changes from `@opensystemslab/planx-document-templates`, make sure to update the commit hash in package.json first